MANIFEST.in

```diff
@@ -1,3 +1,8 @@
 graft habitat/utils/visualizations/assets
```

Comment:
Please verify with https://github.com/mgedmin/check-manifest, you may want to add this a pre-commit hook once it passes.